.github/workflows/codeql.yml

```diff
@@ -0,0 +1,74 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
i don't really know anything about this or how it works...

Reply:
https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/managing-code-scanning-alerts-for-your-repository

according to this its a bit awkward: anyone can see the status on a PR but only an admin can see the list of alerts more generally.